docker-images/testing-python3-wrk-h2load/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM manimaul/xio:base-stretch as builder
+
+# nghttp2 sources
+RUN apt-get install -y wget bzip2 checkinstall
+WORKDIR /build
+RUN wget https://github.com/nghttp2/nghttp2/releases/download/v1.32.0/nghttp2-1.32.0.tar.bz2
+RUN tar -xf nghttp2-1.32.0.tar.bz2
+WORKDIR /build/nghttp2-1.32.0
+
+# nghttp2 build dependencies
+RUN apt-get -y install build-essential libjemalloc-dev libev-dev libevent-dev libjansson-dev libspdylay-dev libssl-dev libxml2-dev zlib1g-dev libc-ares-dev libcunit1-dev libicu-dev libcunit1 libevent-core-2.0-5 libevent-extra-2.0-5 libevent-openssl-2.0-5 libevent-pthreads-2.0-5 icu-devtools libjs-sphinxdoc libjs-underscore docutils-common  pkg-config python-alabaster python-babel python-babel-localedata python-docutils python-imagesize python-jinja2 python-markupsafe python-pygments python-roman python-sphinx python-tz sphinx-common
```

Comment:
Use standard style here, as outlined in [the best-practices doc](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments).